File: Atividades Udemy/Python/secao08/atividade02.py
vetor1 = []
vetor2 = []
soma = []

for n in range(0,10):
    num1 = int(input("Informe um valor para o primeiro vetor: "))
    vetor1.append(num1)
    num2 = int(input("Informe um valor para o segundo vetor: "))
    vetor2.append(num2)
    soma.append(num1 + num2)
for n in soma:
    print(f"O resultado é: {n}")

File: Atividades Udemy/Java/secao08/atividade02.java
package secao08;

import java.util.Scanner;

public class atividade02 {

	public static void main(String[] args) {
		int[] vetor1 = new int[10];
		int[] vetor2 = new int[10];
		int[] soma = new int[10];
		Scanner teclado = new Scanner(System.in);
		
		for (int i = 0; i < 10; i++) {
			System.out.println("Informe o valor do primeiro vetor: ");
			vetor1[i] = teclado.nextInt();
			System.out.println("Informe o valor do segundo vetor: ");
			vetor2[i] = teclado.nextInt();
			soma[i] = vetor1[i] + vetor2[i];
		}
		for (int i = 0; i < 10; i++) {
			System.out.println("O valor da soma dos vetores é: " + soma[i]);
		}
		teclado.close();
	}

}